pkg/minikube/command/ssh_runner.go

```diff
@@ -263,7 +263,7 @@ func (s *SSHRunner) Copy(f assets.CopyableFile) error {
 	mtime, err := f.GetModTime()
 	if err != nil {
 		glog.Infof("error getting modtime for %s: %v", dst, err)
-	} else {
+	} else if mtime != (time.Time{}) {
```

Comment:
In the future, this would probably better as:

`if !mtime.IsZero() { do something }`